Mutagen.Bethesda.Analyzers.Skyrim/Record/Dialog/Responses/PersuasionAmuletConditionAnalyzer.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Mutagen.Bethesda.Analyzers.SDK.Analyzers;
+using Mutagen.Bethesda.Analyzers.SDK.Results;
+using Mutagen.Bethesda.Analyzers.SDK.Topics;
+using Mutagen.Bethesda.Skyrim;
+namespace Mutagen.Bethesda.Analyzers.Skyrim.Record.Dialog.Responses;
+
+public class PersuasionAmuletConditionAnalyzer : IIsolatedRecordAnalyzer<IDialogResponsesGetter>
+{
+    public static readonly TopicDefinition MissingCreatedObject = MutagenTopicBuilder.DevelopmentTopic(
+            "Missing Amulet of Articulation Condition",
+            Severity.Suggestion)
+        .WithoutFormatting("Persuasion check is missing auto pass condition when Amulet of Articulation is equipped");
```

Comment:
Woof.  Breaking my brain.  Definition subjective/opinionated to a specific game setup, but still is helpful and interesting, haha.

Like, what if it's Enderal or some other game that doesn't even have the Amulet of Articulation?   

And why is this item special?   What about other items that are from mods that expect similar things?  Why no analyzers for them?    Sideloads to the rescue?

Reply:
I think Enderal (seeing as it has it's own Mutagen game version) is quite separate and we can assume that people play Skyrim here. But for sure there could be a category of analyzers in the future that are specifically for these kinds of things.
This is special because it is a precedent set by Bethesda. Other mods that add this sort of stuff would need to add their stuff to the formlist which would make them automatically compatible. If they would like to add another way to auto pass skill checks then they would need to adjust the whole base game to be consistent which is not feasible from a compatibility standpoint so nobody does that.